workshop.yml

```diff
@@ -2,15 +2,10 @@
 name: workshop
 channels:
   - conda-forge
-  - bokeh
 
 dependencies:
   - python=3.6
   - pip
-  - bokeh
-  - cartopy
-  - dask
-  - dill
```

Comment:
we need dill otherwise spvcm throws a bunch of warnings when pysal gets imported the first time.

Reply:
oh right. i forgot